src/Jackett.Common/Indexers/milkie.cs

```diff
@@ -0,0 +1,164 @@
+            AddCategoryMapping("6", TorznabCatType.OtherMisc, "App");
+            AddCategoryMapping("7", TorznabCatType.XXX, "Adult");
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+
+            try
+            {
+                var results = await PerformQuery(new TorznabQuery());
+
+                if (results.Count() == 0)
+                    throw new Exception("Testing returned no results!");
+
+                IsConfigured = true;
+                SaveConfig();
+                return IndexerConfigurationStatus.Completed;
+            }
+            catch (Exception e)
+            {
+                IsConfigured = false;
+                throw new ExceptionWithConfigData(e.Message, configData);
+            }
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var queryParams = new NameValueCollection
+            {
+                { "ps", "100" }
+            };
+
+            if (!string.IsNullOrWhiteSpace(query.SearchTerm))
+            {
+                queryParams.Add("query", query.SearchTerm);
+            }
+
+            var endpoint = TorrentsEndpoint + "?" + queryParams.GetQueryString();
+            var jsonResponse = await RequestStringWithCookies(
+                endpoint,
+                null,
+                null,
+                new Dictionary<string, string>() { { "x-milkie-auth", configData.Key.Value } }
+            );
+
+            var releases = new List<ReleaseInfo>();
+
+            try
+            {
+                var response = JsonConvert.DeserializeObject<MilkieResponse>(jsonResponse.Content);
+
+                var dlQueryParams = new NameValueCollection
+                {
+                    { "key", configData.Key.Value }
+                };
+
+                foreach (var torrent in response.Torrents)
+                {
+                    var release = new ReleaseInfo()
+                    {
+                        Title = torrent.ReleaseName,
+                        Link = new Uri($"{TorrentsEndpoint}/{torrent.Id}/torrent?{dlQueryParams.GetQueryString()}"),
+                        Comments = new Uri($"{SiteLink}browse/{torrent.Id}"),
+                        Guid = new Uri($"{SiteLink}browse/{torrent.Id}"),
+                        Size = torrent.Size,
+                        Category = MapTrackerCatToNewznab(torrent.Category.ToString()),
+                        Seeders = torrent.Seeders,
+                        Peers = torrent.Seeders + torrent.PartialSeeders + torrent.Leechers,
+                        Grabs = torrent.Downloaded,
+                        UploadVolumeFactor = 1,
+                        DownloadVolumeFactor = 0,
+                        MinimumRatio = 0,
+                        MinimumSeedTime = 0,
```

Comment:
Are you sure about those values? Those mean freeleech in all torrents, no ratio and no minimum seed time.
I think @garfield69 can help you with that

Reply:
Yeah, it's a new(ish) site, so for now there aren't any rules. I'll update later if things change.

Reply:
It's ok, but that values can be a bit aggressive. Some apps like Sonarr will try to stop the seeding just after downloading. I recommend you these values instead (suggestion):
                        MinimumRatio = 1,
                        MinimumSeedTime = 172800, // 48 hours

Reply:
Alright, will do.